### ISSUE TITLE
Chef/Modernize/CronDFileOrTemplate fails to flag matching resources that use string interpolation

### ISSUE THREAD

**Issue by yacn** [chef/cookstyle#940]
# Version:

7.25.10

# Environment:

* OS: Fedora 35
* cookbook: minimal test case to reproduce (essentially a single file:)

# Scenario:

`Chef/Modernize/CronDFileOrTemplate` should fire in all cases it's relevant.

# Steps to Reproduce:

create a minimal cookbook with the following recipe and run cookstyle. 

```ruby
job = 'fizz'
file "/etc/cron.d/#{job}" do
  action :delete
end
```

# Expected Result:

An offense for `Chef/Modernize/CronDFileOrTemplate` should be raised but is not.

# Actual Result:

cookstyle reports no offenses detected

```
[isaacboehman@fedora tst-cb]$ cookstyle
Inspecting 2 files
..

2 files inspected, no offenses detected
```

**Comment by yacn** [comment]
fixed in #941
